### DISCUSSION TITLE
Is it possible to use certification base authentication as a single authn

### DISCUSSION THREAD

**Discussion by mengleang-ngoun** [JanssenProject/jans#5459] [Ideas]
I have an idea to create a passwordless authn by using a X509 certification

**Comment by ossdhaval** [comment]
Hey @mengleang-ngoun, we are all listening. :ear:   Can you elaborate? :spiral_notepad:

**Reply by mengleang-ngoun** [reply]
I want to implement certification-based authentication means that users can upload their X509 certification that contains user info in that. That certificate is issued by my system.

**Reply by mengleang-ngoun** [reply]
![image](https://github.com/JanssenProject/jans/assets/54835528/1d91db8a-88d3-4d10-99a6-faa61e5aefec)
Based on this diagram above, certification authn this the first step and username/password is the second step in this authn process. But I wan to skip a second. dose it have any security risk. Thank you

**Comment by nynymike** [comment]
Did you check out the design for cert authentication here: https://github.com/GluuFederation/zero-trust-demo/tree/main/jans 
Also `oxAuth` --> `Auth Server` as this is Janssen Project, not Gluu :-)

**Reply by mengleang-ngoun** [reply]
Thank you for sharing this information.

**Reply by mengleang-ngoun** [reply]
Currently, I play around with Janssen with supergluu. But I have a problem with /etc/certs/super_gluu_creds.json. how to we get an access_key and secret_access_key of the Push Notification Server hosted by Gluu.
Thank you supporting us
